pkgs/by-name/li/limine/package.nix

```
@@ -47,7 +47,13 @@ let
     ++ lib.optionals pxeSupport [ "--enable-bios-pxe" ]
     ++ lib.concatMap uefiFlags (
       if targets == [ ] then [ stdenv.hostPlatform.parsed.cpu.name ] else targets
-    );
+    )
+    ++ [
+      "TOOLCHAIN_FOR_TARGET=llvm"
+      # `clang` on `PATH` has to be unwrapped, but *a* wrapped clang
+      # still needs to be available
+      "CC=${stdenv.cc}/bin/clang"
```

`pkgsCross.*.limine` doesn't build because those clang executables are prefixed by the target triple, e.g. `/bin/riscv64-unknown-linux-gnu-clang`
```suggestion
      "CC=${lib.getExe stdenv.cc}"
```
This change would also allow us to build the host tools with gcc I think. Sorry for the late review